odoo/addons/base/models/res_lang.py

```diff
@@ -332,6 +332,16 @@ def format(self, percent, value, grouping=False, monetary=False):
 
         return formatted
 
+    def copy_data(self, default=None):
+        default = dict(default or {})
+
+        if "name" not in default:
+            default["name"] = _("%(lang_name)s (copy)", lang_name=self.name)
+        if "code" not in default:
+            default["code"] = _("%(lang_code)s (copy)", lang_code=self.code)
+        if "url_code" not in default:
+            default["url_code"] = _("%(lang_url_code)s (copy)", lang_url_code=self.url_code)
```

Comment:
We usually translate "(copy)" everywhere else in the code when duplicating a record and I think it's better to since otherwise it might look like gibberish to non-English speaking customers :thinking: It's not a big deal either way though

Reply:
I kept "(copy)" translated but reduced the number of strings to be translated from 3 to 1